.github/workflows/release-prepare-hotfix.yml

```diff
@@ -99,3 +99,44 @@ jobs:
       - name: push changes
         run: |
           git push origin ${{ inputs.ref }}
+
+      - name: fetch develop and main
+        if: ${{ github.event.inputs.tag_version == 'latest' }}
+        run: |
+          git fetch origin develop main
+
+      - name: merge into main
+        if: ${{ github.event.inputs.tag_version == 'latest' }}
+        run: |
+          git checkout main
+          git merge ${{ inputs.ref }} --no-ff
+          git push origin main
+
+      - name: create PR to develop
+        if: ${{ github.event.inputs.tag_version == 'latest' }}
+        env:
+          GITHUB_TOKEN: ${{ steps.generate-token.outputs.token }}
+        run: |
+          git checkout ${{ github.event.inputs.ref }}
+          git checkout -b support/hotfix-merge-conflicts
+          git push origin support/hotfix-merge-conflicts
```

Comment:
imagine for some reason `support/hotfix-merge-conflicts` branch already exist in the remote ?

Reply:
(same as above)